src/nimblepkg/nimscriptsupport.nim

```diff
@@ -200,13 +200,21 @@ proc findNimscriptApi(options: Options): string =
 
 proc getNimPrefixDir(): string = splitPath(findExe("nim")).head.parentDir
 
+when declared(ModuleGraph):
```

Comment:
global because we need to share the state here. 'declared' to make nimble compile with older versions of the compiler's API.